lola.html

```diff
@@ -1236,11 +1285,11 @@ <h3>Handling Move Activity and 'movedTo' property</h3>
 
       <p>
         Mappings of old Actor URLs to new locations listed in Move Activity objects or movedTo properties MAY
-        be cached to make it more efficient to validate "previously" values on objects.  A hypothetical 
+        be cached to make it more efficient to validate `previously` values on objects.  A hypothetical
         cache might attempt to refresh the mapping if it is over 24 hours old, but if it can't be refreshed to 
         maintain the cached value for much longer.  This design would allow mistakes to be corrected (such
         as a 'movedTo' value that simply has the wrong URL, or a change of mind has occurred) while still
```

Comment:
```suggestion
        cache might attempt to refresh the mapping if it is over 24 hours old, but, if it can't be refreshed, to 
        maintain the cached value for much longer.  This design would allow mistakes (such as a 'movedTo'
        value that simply has the wrong URL, or a change of mind has occurred) to be corrected while still
```